src/stats/homerun.ts

```diff
@@ -0,0 +1,39 @@
+import { EventEmitter } from "events";
+
+import type { FrameEntryType, FramesType } from "../types";
+import { positionToHomerunDistance } from "../utils/homerunDistance";
+import type { StatComputer } from "./stats";
+
+export class HomerunComputer extends EventEmitter implements StatComputer<number> {
```

Comment:
I don't see the benefit of this `StatComputer` since the distance would increase by a little bit every frame, and a new event fired every frame. Whereas the only information that we would care about is the final result. I think we should remove this stat computer entirely, and just calculate the distance on `getStadiumStats()` call based on the last frame we've received.